proposals/1/patch-steps.md

```diff
@@ -211,8 +222,54 @@ declare type PatchStepInclude = {
 	// File Path, default protocol "mod:"
 	"src": string;
 };
+
+
+/**
+* The FOR_IN patch step takes a value entry from the values property,
+* goes through the body statements,
+* clones and replaces keyword match with the current value entry inside the statement object,
+* and then executes the current statement.
+**/
+declare type PatchStepForIn = {
+	"type": "FOR_IN";
+	"values": string[] | PatchStepObject[];
+	// interpreted as a Regular Expression
+	"keyword": string | PatchStepObjectMatch;
+	"body": PatchStepsPatch;
+};
+
+/*
+* The COPY patch step takes the Current Value stored in the Interpreter 
+* and maps it to the provided alias key.
```

Comment:
This is adding state to the interpreter, which isn't properly documented in the overview in The Patch Format or in PatchStepMachineState.
Command description itself is fine but should then state where the alias keys are going to/from.